kensanni/More-recipes
# Ft api endpoint for user to sign up using sequelize 152119566

#### What does this PR do
This PR allows a user to sign up with the API signup routes

#### Description of Task to be completed
Allow a user to sign up

#### How should this be manually tested
>- git clone this repo @ https://github.com/kensanni/More-recipes.git
>- then run npm install to run the required modules
>- Ensure you have sequelize, then run sequelize db:migrate to create migration
>- Then test the API endpoint using postman

#### What are the relevant pivotal tracker stories
152119566
